code/game/machinery/igniter.dm

```diff
@@ -57,7 +57,7 @@
 
 /obj/machinery/sparker
 	name = "mounted igniter"
-	desc = "A wall-mounted ignition device."
+	desc = "A wall-mounted ignition device. This mounted igniter has been modified heavily to be able to light plasma in the controlled environment of the Toxin Burner."

```

Comment:
This should be reserved only for varedits

Reply:
their is not way to makes these, and are ONLY used in sci